src/slices/filters.ts

```diff
@@ -61,14 +62,12 @@ const reducers = {
       state.static = [];
     }
     const { filter: targetFilter, shouldSelect } = payload;
-    const foundFilter = state.static.find(storedSelectableFilter => {
+    const matchedFilter = state.static.find(storedSelectableFilter => {
```

Comment:
Minor nit, could we say `matchingFilter`?

Reply:
yup